geode-protobuf/src/test/java/org/apache/geode/protocol/RoundTripCacheConnectionJUnitTest.java

```diff
@@ -291,56 +298,101 @@ public void testConnectionCountIsProperlyDecremented() throws Exception {
   }
 
   @Test
-  public void testNewProtocolRespectsMaxConnectionLimit() throws IOException, InterruptedException {
-    cache.getDistributedSystem().disconnect();
+  public void testNewProtocolRespectsMaxConnectionLimit_notSelector() throws Exception {
```

Comment:
I think it would be much nicer to use a parametrized test for this. I used this in [my PR for ClientHealthMonitor](https://github.com/pivotal-amurmann/geode/blob/8ed3826acb8ad3f6db14889d35e3b8e03a056de1/geode-protobuf/src/test/java/org/apache/geode/protocol/acceptance/CacheConnectionJUnitTest.java)